deploy/charts/rook-ceph/values.yaml

```
@@ -44,6 +44,9 @@ unreachableNodeTolerationSeconds: 5
 # -- Whether the operator should watch cluster CRD in its own namespace or not
 currentNamespaceOnly: false
 
+# -- Pod labels
+podLabels: {}
```

How about if we name this more specific for the operator?
```suggestion
operatorLabels: {}
```

---

operatorLabels is too broad. It could mean the labels on the deployment or the labels on the pod, or both.
But its common practise to separate them.

Check https://github.com/rook/rook/blob/master/deploy/charts/rook-ceph/templates/deployment.yaml and watch for the place where the annotations are injected. Its not on both objects, its on the pod. That behavior its best practise on many helm-chart-projects.

---

I think `podLabels` will be too general because they will think it will apply to all rook pods, not just the operator. What about this?
```suggestion
operatorPodLabels: {}
```